.gitignore

```diff
@@ -23,20 +29,18 @@
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.history/*
 
 # misc
 /.sass-cache
 /connect.lock
 /coverage
 /libpeerconnection.log
 npm-debug.log
+yarn-error.log
 testem.log
 /typings
 
-# e2e
```

Comment:
This could be added back

Reply:
I have removed the `e2e` folder so that's not needed